charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -500,42 +505,21 @@ spec:
           emptyDir: {}
         - name: tmpdir
           emptyDir: {}
+{{- if .Values.providers.openshift.controlPlaneMonitoring }}
+        - name: etcd-config
```

Comment:
In the Operator each check is mounted to its own subdirectory (`/etc/datadog-agent/conf.d/<check>.d/`) but in helm all checks are mounted directly to `/conf.d`, except for `etcd.yaml`, which is added both at `/conf.d/etcd.yaml` and with `etcd-config` to `/conf.d/etcd.d/etcd.yaml`. Is it necessary to duplicate mounting it in two different places?